packages/grafana-ui/src/components/Tooltip/Popover.tsx

```diff
@@ -66,9 +66,8 @@ class Popover extends PureComponent<Props> {
                 >
                   {({ ref, style, placement, arrowProps, update }) => {
                     return (
-                      // TODO: fix keyboard a11y
-                      // eslint-disable-next-line jsx-a11y/no-static-element-interactions
-                      <div
+                      <button
```

Comment:
haven't had time to properly look at this (can look after lunch), but don't think this should be a `button`